tests_for_plate.py

```diff
@@ -0,0 +1,115 @@
+import unittest
+from plate import PlateCell
+from plate import Dimensions
+
+class TestPlateCell(unittest.TestCase):
+    """Tests for PlateCell class"""
+
+    """Tests for 'calculate_row_and_column' method"""
+
+    def test_first_row_and_first_column_are_returned_for_the_first_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(0, dimensions)
+        row_and_column = cell.calculate_row_and_column()
+        self.assertEqual([0, 0], row_and_column)
+
+    def test_sixteenth_row_and_twelfth_column_are_returned_for_the_middle_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(191, dimensions)
+        row_and_column = cell.calculate_row_and_column()
+        self.assertEqual([15, 11], row_and_column)
+
+    def test_sixteenth_row_and_twenty_fourth_column_are_returned_for_the_last_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(383, dimensions)
+        row_and_column = cell.calculate_row_and_column()
+        self.assertEqual([15, 23], row_and_column)
+
+    def test_zero_row_and_zero_column_are_returned_for_0_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(0, dimensions)
```

Comment:
Note that this is actually inconsistent with Tecan (which starts indexing from 1 for cell numbers - not from 0). You may decide to still work with 0-based indices anyway. We usually keep our internal representation in whatever form we like and then transform to the external (Tecan) format at the very end (e.g. when generating worklists).